src/docOpts.ts

```diff
@@ -4,6 +4,9 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
is this rule enabled in the `tsconfig`? why do we have to add this to disable it in this file?

Reply:
it's in the current eslint rules.  I struggled to understand some of this code and didn't want to change things I didn't understand.